source/content-modelling/how-embedding-works/index.html.md.erb

```diff
@@ -0,0 +1,93 @@
+---
+title: How embedding works
+weight: 4
+layout: multipage_layout
+---
+
+# How embedding works
+
+Content Block Embedding is mainly handled in the [Publishing API](https://github.com/alphagov/publishing-api/), with some shared code in the
+[Content Block Tools gem](https://github.com/alphagov/govuk_content_block_tools) detailed in the diagram and steps below:
+
+<pre lang="mermaid">
```

Comment:
should I be able to see this as a diagram on the frontend? or is it something I need to plug in elsewhere?

Reply:
(cool idea if we can get to see the diagram - I've only ever made fiddly sequence diagrams by hand so this would be great!)

Reply:
Yeah, it should work. I added the extra JS needed in https://github.com/alphagov/govuk-developer-docs/pull/5019

Reply:
Here's a screenshot:

![image](https://github.com/user-attachments/assets/7498578e-2dfe-45f2-8ce4-a84f089e0786)

Reply:
Ah yeah, I'll sort that 👍